server/external/rapidapiClient.js

```diff
@@ -1,8 +1,78 @@
 const axios = require("axios").default;
+const { checkCache, getCachedCall, cacheData } = require('../redis.repository');
+
+
+const getFromCache = async (requestUrl) => {
+	try {
+        const dataString = await getCachedCall(requestUrl);
+        if (dataString !== null){
+            const data = JSON.parse(dataString);
+            return data;
```

Comment:
I think we can just return `JSON.parse(dataString)` here without assigning it as a variable. Let me know what you think!

Reply:
Good catch, I agree that would be cleaner